src/components/StoriesList/Pagination.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import { usePathname, useSearchParams } from "next/navigation";
+
+interface Props {
+  total_pages: number;
+}
+
+export default function Pagination({ total_pages }: Props) {
+  const params = Object.fromEntries(useSearchParams().entries());
+  if (!params.page) {
+    params.page = "1";
+  }
+  const page_number = parseInt(params.page);
+  const pages = Array.from({ length: total_pages }, (_, i) => i + 1);
+
+  return (
+    <>
+      <ul className="inline-flex justify-center gap-4 align-middle text-lg text-sciquelMuted">
+        <li>Page</li>
+        {page_number !== 1 && (
+          <li>
+            <a
+              className=""
```

Comment:
Empty className?

Reply:
I was experimenting tailwind css class and forgot to remove this line. Removing now...